app/static/dayview.css

```diff
@@ -72,20 +72,20 @@
 }
 
 .total-time {
-  font-size: 0.8rem;
+  font-size: var(--text_s);
 }
 
-.title_size_small {
-  font-size: 0.8em;
+.title-size-small {
+  font-size: var(--text_m);
 }
 
-.title_size_Xsmall {
-  font-size: 0.4em;
+.title-size-Xsmall {
```

Comment:
Last issue here. "title-size-Xsmall" -> "title-size-x-small", all letters should be small.